packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -183,7 +183,7 @@ Documentation for %{pkg_name}.
 gem unpack %{SOURCE0}
 %{?scl:EOF}
 
-%setup -q -D -T -n  %{gem_name}-%{version}
+%setup -q -D -T -n  %{gem_name}-%{version}%{prever}
```

Comment:
I led you astray, drop this prever addition here

Reply:
This definitely got the build further so something else probably needs to change higher up to accompany the revert?